### DISCUSSION TITLE
Bisq scam?

### DISCUSSION THREAD

**Discussion by alturn97** [bisq-network/bisq#6153] [General]
Yesterday I posted an ad on Bisq, I was asked to deposit a deposit, of 0.00182156 BTC, the funds were blocked, for the order, however today I saw these two outgoing transactions, wasn't my wallet multisign? Also I am sure my private keys are not compromised, so should I think bisq is a scam? Can anyone help me out?!

![image](https://user-images.githubusercontent.com/104059432/164176571-65ed97fe-a1a0-4ce2-ba48-443b1a357c94.png)

bc1q0yjmkyuphv5j7n4r5v3x7uvk4230wre62w55pyg0uegkq8rg268sup4aef

This is the address my BTCs are now at

**Comment by ghost** [comment]
This was answered on matrix, user has imported seedphrase into another instance of Bisq.
